apps/admin-ui/src/component/Navigation.tsx

```diff
@@ -1,131 +1,229 @@
+  const isActive = exact ? pathname === href : pathname.startsWith(href);
+
+  return (
+    <Header.ActionBarSubItem
+      key={href}
+      href={`/kasittely/${href}`}
+      label={t(title)}
+      className={isActive ? "active" : ""}
+      notificationBubbleAriaLabel={shouldDisplayCount ? "Määrä" : undefined}
+      notificationBubbleContent={
+        shouldDisplayCount ? count.toString() : undefined
+      }
+    />
+  );
+}
+const Navigation = ({ apiBaseUrl }: Props) => {
+  const { t } = useTranslation();
   const { user } = useSession();
   const firstName = user?.firstName?.trim() ?? "";
   const lastName = user?.lastName?.trim() ?? "";
   const name = `${firstName} ${lastName}`.trim() || t("Navigation.noName");
+  const { handlingCount, hasOwnUnits } = useHandling();
+  const { hasSomePermission, user: userPermission } = usePermission();
+  if (!userPermission) return null;
 
-  return (
-    <StyledHDSNavigation
-      theme={{
-        "--header-background-color":
-          "var(--tilavaraus-admin-header-background-color)",
-        "--header-color": "var(--tilavaraus-admin-header-color)",
-      }}
-      title={t("common.applicationName")}
-      menuToggleAriaLabel="Menu"
-      skipTo="#main"
-      skipToContentLabel={t("Navigation.skipToMainContent")}
-      onTitleClick={onLogoClick}
-      onMenuToggle={() => setMenuState(!isMenuOpen)}
-      menuOpen={isMenuOpen}
-    >
-      <HDSNavigation.Actions>
-        <MobileNavigation>
-          {user && !disabledRouter && (
-            <MainMenu
-              placement="navigation"
-              onItemSelection={() => setMenuState(false)}
-            />
-          )}
-        </MobileNavigation>
-        <UserMenu
-          userName={name}
-          authenticated={user != null}
-          label={t(user != null ? "Navigation.logging" : "Navigation.login")}
-          onSignIn={() => signIn(apiBaseUrl)}
-        >
-          {user && (
-            <UserInfo
-              name={name}
-              email={user?.email || t("Navigation.noEmail")}
-            />
-          )}
-          <HDSNavigation.Item
-            className="btn-logout"
-            label={t("Navigation.logout")}
-            onClick={() => signOut(apiBaseUrl)}
-            variant="primary"
-          />
-        </UserMenu>
-      </HDSNavigation.Actions>
-    </StyledHDSNavigation>
+  const menuItemList = getFilteredMenu(hasOwnUnits, hasSomePermission).filter(
+    (item) => item != null
   );
-};
 
-// NOTE requires both client and react-router context
-const NavigationWithRouter = ({ apiBaseUrl }: Pick<Props, "apiBaseUrl">) => {
-  const history = useNavigate();
   return (
-    <Navigation apiBaseUrl={apiBaseUrl} onLogoClick={() => history("/")} />
+    <Header>
+      <ActionBar
+        title={t("common:applicationName")}
+        titleAriaLabel={t("common:applicationName")}
+        frontPageLabel={t("common:gotoFrontpage")}
```

Comment:
this is missing in https://github.com/City-of-Helsinki/tilavarauspalvelu-ui/blob/0325933f0d65eabd92834907ef93162e22cc814e/apps/admin-ui/src/i18n/index.ts#L21

need to add the same translations to `admin-ui` as `ui`

Reply:
Removed the Footer entirely from `admin-ui`